stack.yaml

```diff
@@ -0,0 +1,66 @@
+# This file was automatically generated by 'stack init'
+#
+# Some commonly used options have been documented as comments in this file.
+# For advanced use and comprehensive documentation of the format, please see:
+# https://docs.haskellstack.org/en/stable/yaml_configuration/
+
+# Resolver to choose a 'specific' stackage snapshot or a compiler version.
+# A snapshot resolver dictates the compiler version and the set of packages
+# to be used for project dependencies. For example:
+#
+# resolver: lts-3.5
+# resolver: nightly-2015-09-21
+# resolver: ghc-7.10.2
+# resolver: ghcjs-0.1.0_ghc-7.10.2
+# resolver:
+#  name: custom-snapshot
+#  location: "./custom-snapshot.yaml"
+resolver: lts-10.2
```

Comment:
Everything in this file except this line can be safely removed. Let's not keep trash in repository :+1: :green_heart: 
Also, latest LTS is 11.4, I think you can safely bump up it.

Reply:
I use intellij-haskell plugin, which does not support LTS 11.x yet.
[Issue with LTS 11.x](https://github.com/rikvdkleij/intellij-haskell/issues/266)

Reply:
Okay, good to know, It's not critical.